indexer/src/test/java/org.pdxfinder/commands/BaseTest.java

```diff
@@ -0,0 +1,24 @@
+package org.pdxfinder.commands;
```

Comment:
The actual path of this file is strange.  It seems to include a "." which should be a "/"

Reply:
Yeah it's strange, I figured it was just a java quirk...

Reply:
Should refactor to change the path.  I'll resolve the related comments as they are all in the same package.

Reply:
In the file structure it is a "/" - looks like both intellj and github replace with "." when the dir is empty

Reply:
Yeah, I think a DIRECTORY with name "org.pdxfinder" was created rather than a PACKAGE with name "org.pdxfinder"

Reply:
This should now be addressed, thanks @sprintell